Makefile

```diff
@@ -0,0 +1,19 @@
+IMAGE_REGISTRY ?= quay.io/kubevirt
+MUST_GATHER_IMAGE ?= must-gather
+NODE_GATHER_IMAGE ?= node-gather
+IMAGE_TAG ?= latest
+
+build: manifests docker-build docker-push
+
+manifests: 
+	./build/release-manifests.sh ${IMAGE_REGISTRY}
+
+docker-build: 
+	docker build -t ${IMAGE_REGISTRY}/${MUST_GATHER_IMAGE}:${IMAGE_TAG} .
```

Comment:
why docker and not podman ?

Reply:
Because Travis does not have podman.

Reply:
Haven't tested it myself, but seems replacement is doable with:

http://www.lulf.no/technical/kubernetes/podman/travis/ci/2019/01/31/podman-on-travis.html

Reply:
I've previously used podman/buildah and it worked fine: https://github.com/mmazur/kubevirt-containerized-cloud-image-builder/blob/master/.travis.yml

Reply:
Well, yes you can add additional repositories to install it from I guess... I think I prefer_ native repos, less chance of service disruption when networking goes bad.